pkg/deploy/externaldns/dnsendpoint_manager.go

```diff
@@ -36,11 +39,11 @@ func (s *defaultDnsEndpointManager) Create(ctx context.Context, service *lattice
 		Name:      service.Spec.Name + "-dns",
 	}
 	if service.Spec.CustomerDomainName == "" {
-		glog.V(2).Infof("Skipping creation of %s: detected no custom domain", namespacedName.String())
+		s.log.Debugf("Skipping creation of %s: detected no custom domain", namespacedName.String())
```

Comment:
%s will use Stringer interface and call String(), no need to explicitly call String()

Reply:
Thanks, will include this in part 3!